### ISSUE TITLE
Check if summary at start works

### ISSUE THREAD

**Issue by koppor** [InAnYan/jabref#96]
1. User has new JabRef. This can be achieved with a virtual vm (see https://github.com/JabRef/jabref/tree/main/scripts/vms) or [Windows Sandbox](https://learn.microsoft.com/de-de/windows/security/application-security/application-isolation/windows-sandbox/windows-sandbox-overview)
2. User opens `Chocolate.bib` from our OneDrive (which has some PDFs attached)
3. User clicks on "Corti_2009"
4. User agrees on Privacy Policy (see https://github.com/InAnYan/jabref/issues/92)
5. User waits 
6. User clicks on "AI chat"
7. User sees "Please wait", too

I think, the user should not wait for more than 5 minutes - there is something wrong.

**Comment by ThiloteE** [comment]
Just for context: Corti_2009 has 33 pages and that's roughly 100 000 characters. On reddit somebody said 4 tokens for 3 words on average. That would mean roughly 75 000 tokens. So that's actually quite a lot of tokens.

More information is needed: What AI service are you subscribing to? What model?
What commit in what pull-request?

Some hypotheses for debugging:
- LLM service needs to re-calculate context alot, because the paper is above its supported context window size.
- Something during sending takes too long.
- Does the API service have an outage or rate-limit?
- JabRef tries to do embeddings at the same time. Once for ai chat and once for ai summarization.

**Comment by koppor** [comment]
It was OpenAI and gpt4o.

Independent of that, it is good UI practice to provide progress. German explanation: http://www.usabilitypatterns.info/catalog/patterns/fortschrittsanzeige.html

If one cannot report progress, one should output the expected waiting time.

In case the http connection is lost (typical timeout 1 minute) should also be handled. (I assume, it is, but we need to check)

**Comment by InAnYan** [comment]
I'm not sure, what this issue talks about?

Is this a general issue for progress indication?

**Comment by koppor** [comment]
The issue covers multiple things.

1. progress indication -> I created a new issue https://github.com/InAnYan/jabref/issues/115.
2. A systematic final check before merging the feature. 

Regarding 2:

One can approach this very structured using [System Test Portal](https://www.systemtestportal.org/) or other software. I think, this is too much for our team size. Thus, I created "final-test" label to make it clear, that this should happen if everything is settled. - I also moved the issue as last in the milestone view https://github.com/InAnYan/jabref/milestone/3
